updater/config/config.go

```diff
@@ -0,0 +1,138 @@
+package config
```

Comment:
The main package is supposed to be a thin wrapper around the updater library (due to the existence of onlineconf-csi-driver) and all it has to do is parse the configuration and pass it to updater.NewUpdater. Therefore, I think that it is undesirable to move the work with the config into a separate package.